webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -442,12 +413,12 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     });
     setSelectAll(false);
 
-    // Refetch experiment list to get updates based on batch action.
+    // Re-fetch experiment list to get updates based on batch action.
     await fetchExperiments();
   }, [fetchExperiments, setSelectAll, setSelection]);
 
-  const handleUpdateExperimentList = useCallback(
-    (action: BatchAction, successfulIds: number[]) => {
+  const handleActionSuccess = useCallback(
+    (action: ExperimentAction, successfulIds: number[]) => {
```

Comment:
shouldn't `BatchAction` be a strict subset of `ExperimentAction` here?

Reply:
`handleActionSuccess` handles both the table batch action along with the context menu actions. They're nearly identical except that the successful ids are either multiple ids for batch actions or just a single id for context menus. So generalized the action to `ExperimentAction` to be able to support both.